packages/OverflowMenu/src/OverflowMenu.js

```diff
@@ -108,7 +112,7 @@ function OverflowMenu(props) {
     } else if (event.key === "Enter" || event.key === " ") {
       // do nothing
```

Comment:
is weird to have an if statement with an empty body 🤔 isn't?

Reply:
I think it's ok because it's purpose is to not go into the `else` block.